api/datadoghq/v1alpha2/datadogpodautoscaler_types.go

```diff
@@ -99,6 +108,18 @@ const (
 	DatadogPodAutoscalerApplyModePreview DatadogPodAutoscalerApplyMode = "Preview"
 )
 
+// DatadogPodAutoscalerFallbackMode specifies if the controller should apply/generate recommendations in fallback mode.
+// +kubebuilder:validation:Enum:=Apply;Disabled
+type DatadogPodAutoscalerFallbackMode string
```

Comment:
Wondering if we want to keep the `Mode` if it's only enabled/disabled

Reply:
Good point, I changed it to `Enabled  bool`  instead! (unless we foresee wanting to add another mode in the future?)